tests/api_data/endpoint_data.json

```diff
@@ -177,7 +177,7 @@
       },
```

Comment:
Sorry about this... hope this didn't cause too much trouble fixing.  I'd forgotten to update local data it seems after testing the actual integration

Reply:
no worries 👍